config/settings.yml

```diff
@@ -1446,6 +1446,49 @@ mobile_lighthouse_letters:
   api_scopes:
     - "letters.read"
 
+mobile_lighthouse_benefits_claims:
+  client_id: '0oal917lvbYQbkaQu2p7'
+  key_path: ~
+  aud_claim_url: "https://deptva-eval.okta.com/oauth2/ausdg7guis2TYDlFe2p7/v1/token"
+  access_token_url: 'https://sandbox-api.va.gov/oauth2/claims/system/v1/token'
+  api_url: 'https://sandbox-api.va.gov/services/claims/v1/'
+  api_scopes:
+    - "claim.read"
+    - "claim.write"
+
+mobile_lighthouse_letters_2:
+  client_id: '0oal917lvbYQbkaQu2p7'
+  key_path: ~
+  aud_claim_url: "https://deptva-eval.okta.com/oauth2/ausdg7guis2TYDlFe2p7/v1/token"
```

Comment:
This audience claim assertion url is incorrect.

https://sandbox-api.va.gov/oauth2/va-letter-generator/system/v1/.well-known/openid-configuration

It should be the .well-known issuer + the intended operation, in this case `/v1/token`.